packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1535,6 +1537,13 @@ abstract class FlutterCommand extends Command<void> {
       }
     }
 
+    final Iterable<Device> networkDevices = devices.where((Device device) {
+      return device is IOSDevice && device.interfaceType == IOSDeviceConnectionInterface.network;
+    });
+    if (networkDevices.isNotEmpty) {
+      UsageEvent('device', 'ios-network-device', label: name, flutterUsage: globals.flutterUsage).send();
+    }
```

Comment:
This usage would tell us an absolute number of wireless iOS devices, however I think it would be more interesting to know what percentage of iOS devices are wireless. How about something like:

```suggestion
    final Iterable<IOSDevice> iosDevices = devices.whereType<IOSDevice>();
    if (iosDevices.isNotEmpty) {
      final bool includesNetworkDevices =
          iosDevices.any((IOSDevice device) => device.interfaceType == IOSDeviceConnectionInterface.network);
      UsageEvent(
        'device',
        'ios-interface-type',
        label: includesNetworkDevices ? 'wireless' : 'usb',
        flutterUsage: globals.flutterUsage,
      ).send();
    }
```